systems/robotInterfaces/QPLocomotionPlan.m

```diff
@@ -131,6 +133,7 @@
       else
         supp_idx = find(obj.support_times<=t_plan,1,'last');
```

Comment:
@rdeits FYI, this is "support_times<=t_plan" in mitdrc/drake and "support_times<t_plan" in loco/drake